packages/components/src/BrandMoment/BrandMoment.module.css

```diff
@@ -38,67 +34,73 @@ $breakpoint-xl: 1366px; // TODO: create an xl breakpoint in design-tokens and re
   --brand-moment-background-color: var(--color-red-100);
 }
 
+/*
 // --------------------------------
 // Reusable container
 // --------------------------------
+*/
 .container {
-  max-width: $layout-content-max-width;
+  max-width: var(--layout-content-max-width);
   margin: 0 auto;
-  padding: 0 $spacing-md;
+  padding: 0 var(--spacing-24);
 }
 
-@media (min-width: $layout-breakpoints-large) {
+@media (width >= 1080px) {
   .container {
-    padding: 0 $spacing-xxl;
+    padding: 0 3.75rem;
   }
 }
 
-@media (min-width: $breakpoint-xl) {
+@media (width >= 1366px) {
   .container {
-    padding: 0 $spacing-xxxxl;
+    padding: 0 5.25rem;
```

Comment:
It's possibly a little confusing since it's not a proper spacing token migration guide, but this has been approved by Jess for the old spacing tokens to the new ones.

https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/4459692600/KAIO+Adoption+Guide#Box-migration

You'll need to do a little extra step to get the conversion (Box was using another size system), but essentially grab the old value (in rem) then *16 (to get px), then use the equivalent new token.
In your case, `spacing-xxxxl` = `5.25rem` = `n * 16` = `84px` = `spacing-80`.

You will get updates in Chromatic when you make these changes - these are to be expected, and you'll want to communicate these to consumers too on release.